soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.series

```diff
@@ -14,7 +14,8 @@ config SOC_SERIES
 
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	int
-	default 32768
+	default 32768 if RTC_AS_TIMER
```

Comment:
This could work, but why not using, instead, the existing options:
NRF_RTC_TIMER and CORTEX_M_SYSTICK?